### PR TITLE
Yessenia Rodriguez js arrays

### PR DESCRIPTION
**Pull request by yesseniarod**
Screen Shot 2021-01-25 at 5 32 59 PM

### CODE REVIEW

File: javascript-arrays/main.js [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+colors[2] = 'green';
+console.log('Mexico is ' + colors[0] + ', ' + colors[1] + ', and ' + colors[2]);
+
+console.log('colors:', colors);
+
+var students = ['Jordan', 'Gabby', 'Barbara', 'Marz'];
+var numberOfStudents = students.length;
+
+console.log('There are ' + numberOfStudents + ' students in the class.');
+var lastStudent = students[3];
```
Comment (codingcodymiller) [inline javascript-arrays/main.js:17]: Screen Shot 2021-01-25 at 5 38 51 PM

**Review by codingcodymiller — CHANGES_REQUESTED** [review]
Hey Yessenia! One change, please see the screenshot for what adjustments need to be made

File: javascript-arrays/main.js (hunk 1 above)
Comment (yesseniarod) [reply javascript-arrays/main.js]: Screen Shot 2021-01-25 at 6 06 00 PM

**Review by codingcodymiller — APPROVED** [review]
Nice work!!! 🎉

**Pull request merged**